packages/blue-sdk/src/addresses.ts

```diff
@@ -363,9 +368,9 @@ export const addressesRegistry = {
   },
 } as const;
 
-export const addresses = addressesRegistry as Record<number, ChainAddresses>;
+export const addresses = _addressesRegistry as Record<number, ChainAddresses>;
```

Comment:
This reference is constant but should not be because it's used in many places (and may be used by integrators)